couchbase/tests/test_couchbase.py

```diff
@@ -34,6 +35,9 @@
     'vb_replica_curr_items',
 ]
 
+if COUCHBASE_MAJOR_VERSION == 7:
+    NODE_STATS += ['index_data_size', 'index_disk_size']
```

Comment:
nit- can you make a variable in `common.py` for ['index_data_size', 'index_disk_size'] since it's used twice?